app/Http/Controllers/PostsController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Post;
+use DB;
+
+class PostsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+
+    // index => listing of all the posts
+    // create => represent form
+    // store => submit to the function , take care of submitting it to the model to the database
+    // edit => edit form
+    // update
+    // delete
+    // show => showing a single post
+
+    public function index()
+    {
+        //fetch all of the data in the model in the table
+        // $posts = Post::all(); // return this line and stops. not loading the view
```

Comment:
There's no reason to comment codes. If it's not valid anymore, you can remove it (and let VCS keep the history)

Reply:
It helps me to memorize different format of doing One thing in purpose. Is it wrong?